tests/convolve.py

```diff
@@ -173,26 +179,28 @@ def sameMaskPlaneDicts(maskedImageA, maskedImageB):
 
 class ConvolveTestCase(utilsTests.TestCase):
     def setUp(self):
-        self.maskedImage = afwImage.MaskedImageF(FullMaskedImage, InputBBox, afwImage.LOCAL, True)
-        # use a huge XY0 to make emphasize any errors related to not handling xy0 correctly.
-        self.maskedImage.setXY0(300, 200)
-        self.xy0 = self.maskedImage.getXY0()
-
-        # provide destinations for the convolved MaskedImage and Image that contain junk
-        # to verify that convolve overwrites all pixels;
-        # make them deep copies so we can mess with them without affecting self.inImage
-        self.cnvMaskedImage = afwImage.MaskedImageF(FullMaskedImage, ShiftedBBox, afwImage.LOCAL, True)
-        self.cnvImage = afwImage.ImageF(FullMaskedImage.getImage(), ShiftedBBox, afwImage.LOCAL, True)
-
-        self.width = self.maskedImage.getWidth()
-        self.height = self.maskedImage.getHeight()
-#         smask = afwImage.MaskU(self.maskedImage.getMask(), afwGeom.Box2I(afwGeom.Point2I(15, 17), afwGeom.Extent2I(10, 5)))
-#         smask.set(0x8)
+        if dataDir is not None:
+            self.maskedImage = afwImage.MaskedImageF(FullMaskedImage, InputBBox, afwImage.LOCAL, True)
+            # use a huge XY0 to make emphasize any errors related to not handling xy0 correctly.
+            self.maskedImage.setXY0(300, 200)
+            self.xy0 = self.maskedImage.getXY0()
+    
+            # provide destinations for the convolved MaskedImage and Image that contain junk
+            # to verify that convolve overwrites all pixels;
+            # make them deep copies so we can mess with them without affecting self.inImage
+            self.cnvMaskedImage = afwImage.MaskedImageF(FullMaskedImage, ShiftedBBox, afwImage.LOCAL, True)
+            self.cnvImage = afwImage.ImageF(FullMaskedImage.getImage(), ShiftedBBox, afwImage.LOCAL, True)
+    
+            self.width = self.maskedImage.getWidth()
+            self.height = self.maskedImage.getHeight()
+    #         smask = afwImage.MaskU(self.maskedImage.getMask(), afwGeom.Box2I(afwGeom.Point2I(15, 17), afwGeom.Extent2I(10, 5)))
+    #         smask.set(0x8)
```

Comment:
ConvolveTestCase includes a number of tests that don't require afwdata (eg. runBasicTest, runStdTest etc...). If I used skipIf on the setUp or on the whole class, they would also gety skipped. This way, only parts of ConvolveTestCase get skipped.